README.md

```diff
@@ -54,8 +54,8 @@ example project.
 -export([button/1]).
 
 %% Libs.
-%% `live_view.hrl` contains the ?LV macro.
--include_lib("arizona/include/live_view.hrl").
+%% `live_view.hrl` contains the ?ARIZONA_LIVEVIEW macro.
+-include("arizona.hrl").
```

Comment:
It's a "real world" example, so the consumer should use it as an `include_lib` on its app: 
```suggestion
-include_lib("arizona/include/arizona.hrl").
```
Or am I wrong? Would it work with just an include?

Reply:
When it was in the context of the application itself it would work, but for a consumer it wouldn't. I can change it, but I have plans to revisit the README before 1.0.0 😄

Reply:
3645a99 (auto-resolved by accepting the suggestion)